java/src/main/kotlin/io/spine/validation/java/GenerationContext.kt

```diff
@@ -134,6 +162,14 @@ internal constructor(
      */
     val isElement: Boolean = elementReference != null
 
+    init {
+        System.err.println(TypeSystem::class.memberProperties.find { it.name == "files" }!!.let {
```

Comment:
Why are we having this console output here? If it's really needed, can we have it as a method of a companion with appropriate documentation?